go/api/generic_commands.go

```diff
@@ -30,4 +30,30 @@ type GenericBaseCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/del/
 	Del(keys []string) (Result[int64], error)
+
+	// Copy copies the value stored at the source key to the destination key.
+	//
+	// By default, the destination key is created in the same logical database used by the connection. However, the `DB` option allows specifying an alternative logical database index for the destination key. If the destination key already exists, the command returns zero. The `REPLACE` option can be used to remove the destination key before copying the value.
+	//
+	// Note:
+	// - When in cluster mode, the command may route to multiple nodes depending on the location of the source and destination keys.
```

Comment:
when in cluster, both keys should be managed by the same node.

Please use another notice.